infra/snd/01/nginx-ingress-values.yaml

```diff
@@ -11,6 +11,7 @@ spec:
     controller:
       service:
         annotations:
+          service.beta.kubernetes.io/azure-pls-visibility: ""
```

Comment:
Is it empty value or to feed later?

Reply:
this shouldn't be visible to other subscriptions outside of our core subs right?

Reply:
@ganeshg2491  that will be empty.